server/build/plugins/json-pages-plugin.js

```diff
@@ -0,0 +1,24 @@
+export default class JsonPagesPlugin {
+  apply (compiler) {
+    compiler.plugin('after-compile', (compilation, callback) => {
+      const pages = Object
+        .keys(compilation.assets)
+        .filter((filename) => /^bundles\/pages.*\.js$/.test(filename))
```

Comment:
This regex doesn't match the way one could wish for in windows, causing pages not to be compiled to json, causing issue #597 thus making most tests fail on windows.